docs/ossm/istioctl/README.md

```diff
@@ -0,0 +1,81 @@
+[Return to OSSM Docs](../)
+
+# Installing the istioctl tool
+
+The `istioctl` tool is a configuration command line utility that allows service 
+operators to debug and diagnose Istio service mesh deployments.
+
+## Prerequisites
+
+* Use an `istioctl` version that is the same version as the Istio control plane 
+for the Service Mesh deployment. For checking the `Istio` version, run the following command:
+
+    ```bash
+    oc -n istio-system get istio
+    ```
+
+## Steps
+
+1. Download `istioctl` binary
+
+    In the OpenShift console, navigate to the Command Line Tools by clicking :grey_question: -> **Command Line Tools** in the upper-right of the header.  
+    Then click on **Download istioctl** and choose the right architecture according to your system.
+
+    **NB**: All the releases of `istioctl` are directly downloadable [here](https://mirror.openshift.com/pub/cgw/servicemesh/)
```

Comment:
Just one question, what does it mean NB?

Reply:
Maybe you could consider using https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts